x-pack/metricbeat/module/ibmmq/qmgr/_meta/testdata/ibmmq-status.9.1.4.0-r1-amd64.plain-expected.json

```diff
@@ -1,7 +1,7 @@
 [
     {
         "event": {
-            "dataset": "ibmmq.qmgr",
+            "dataset": "prometheus",
```

Comment:
Oh wait, I don't like this change, I didn't fully get the implications of the namespace changes in prometheus collector.

It was intentional to have the name of the metricset in `event.dataset`. This way we have all the prometheus metrics under `prometheus.metrics`, but we can still use `event.dataset` to know the source of the metrics.

Reply:
> So I would consider it safe to change it since there are still `event.module` and `metricset.name` which can be also used.

Umm, this could be a breaking change if something or someone is using `event.dataset` for this purpouse (I would have used this field myself, and I have probably recommended it in the past).